hash.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
```

Comment:
[all the usings](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS176209.md#CUSTOM01) 9 times
Flag all usings just for fun